charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml

```diff
@@ -136,14 +138,14 @@ data:
 
     [FILTER]
         Name                parser
-        Match               kubernetes.*kube-controller-manager*kube-controller-manager*
+        Match               kubernetes.*kube-apiserver*vpn-seed*
         Key_Name            log
-        Parser              kubeapiserverParser
+        Parser              vpnshootParser
         Reserve_Data        True
 
     [FILTER]
         Name                parser
-        Match               kubernetes.*kube-scheduler*cloud-kube-scheduler*
```

Comment:
Please, bring back this filter.

Reply:
This filter is there. Just was duplicated.

Reply:
True. I had to open the full file, not just checking the diff provided here.